### PR TITLE
[subzero] prepare canary network for launch

### PR DESCRIPTION
**Pull request by vayesy**
Remove GameDAO pallets.
This is basically copy of `zero` runtime, adapted for Kusama relay chain.
There will also be another PR to setup `subzero-dev` runtime, which includes everything we use on dev.

### CODE REVIEW

File: bin/subzero/runtime/src/lib.rs [hunk 1]
```diff
@@ -158,7 +162,7 @@ impl_opaque_keys! {
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("subzero"),
-	impl_name: create_runtime_str!("dev"),
+	impl_name: create_runtime_str!("live"),
```
Comment (2075) [inline bin/subzero/runtime/src/lib.rs:165]: subzero, not live

Comment (vayesy) [reply bin/subzero/runtime/src/lib.rs:165]: Change `live` to `subzero`?

```rust
pub const VERSION: RuntimeVersion = RuntimeVersion {
	spec_name: create_runtime_str!("subzero"),
	impl_name: create_runtime_str!("subzero"),
	authoring_version: 75,
	spec_version: 63,
	impl_version: 0,
	apis: RUNTIME_API_VERSIONS,
	transaction_version: 1,
	state_version: 1,
};```

Something like this? So `spec_name` and `impl_name` refer to the same value?

**Review by 2075 — APPROVED** [review]
are we replacing our prior dev chain with this setup iirc?

**Comment by vayesy** [comment]
> are we replacing our prior dev chain with this setup iirc?

I would say we do next:
1. rename development version of the chain (known earlier as `subzero`) to `subzero-dev`
2. create new runtime to run on Kusama relay chain and name it `subzero`

**Pull request merged**